custom/opm/incentive.py

```diff
@@ -38,23 +40,23 @@ def __init__(self, worker, report, case_sql_data=None, form_sql_data=None):
         self.block = worker.get('block')
         self.owner_id = worker.get('doc_id')
 
-        if case_sql_data:
-            self.women_registered = len(case_sql_data)
-            self.children_registered = sum([c.num_children for c in case_sql_data if not c.is_secondary])
+        if case_data:
+            self.women_registered = len(case_data)
+            self.children_registered = sum([c.num_children for c in case_data if not c.is_secondary])
+            self.service_forms_count = 'yes' if sum([
+                len(opm_case.filtered_forms(VHND_XMLNS)) for opm_case in case_data]) else 'no'
```

Comment:
That's odd, I didn't realize this was a "yes" or "no" column.  I'll ping Anushree to see about updating the description, but your change is consistent with the current behavior.